### PR TITLE
use correct way of initialization in test case

### PR DESCRIPTION
**Pull request by gxyd**
## Description

Fixes https://github.com/lfortran/lfortran/issues/5341 (considering that issue is valid)

### CODE REVIEW

File: tests/fixedform_doloop3.f [hunk 1]
```diff
@@ -4,7 +4,8 @@ SUBROUTINE START1(N, F, L, LOUT)
 C
 C       GENERATES A 1,N ANSARI-BRADLEY DISTRIBUTION IN F.
 C
-      REAL F(L), ONE=1, TWO=2
```
Comment (Pranavchiku) [inline tests/fixedform_doloop3.f:7]: My question is why `ONE=1` incorrect? because is shall be `one=1.0`? if so, then let's use that instead.

Comment (gxyd) [reply tests/fixedform_doloop3.f:7]: No, the issue is with the syntax, i.e. we can't have `REAL F(L), ONE=1, TWO=2` as the correct syntax is: `REAL :: F(L), ONE=1, TWO=2` (i.e. use of `::`) or as is done currently by use of `data` statement, the only reason to use the data statement is as that's what's used in the original test case which this is being updated to.

Please also see the linked issue.

Comment (certik) [reply tests/fixedform_doloop3.f:7]: We need to double check with GFortran, but I think so.

Comment (gxyd) [reply tests/fixedform_doloop3.f:7]: Yes, I initially mentioned this in the linked issue.

Comment (gxyd) [reply tests/fixedform_doloop3.f:7]: I'll let one of you resolve the conversation if this PR looks good.

**Review by certik — APPROVED** [review]
I think this is correct.

| Fixed-form | Free-form |
| --- | --- |
| integer x | integer :: x |
| integer x = 5 | integerx = 5 |
| integer :: x = 5 | integer :: x = 5 |

These are not allowed in declarations:
```
integer A(10), x = 5
integer x = 5
```

**Pull request merged**